src/apps/jewelswap/constants.tsx

```diff
@@ -0,0 +1,14 @@
+import DiamondIcon from '@mui/icons-material/Diamond';
+import { NFTMarketplace } from './types';
+import xSpotlightLogo from '../../assets/img/jewelswapLogo.png';
+
+export const marketplaces: NFTMarketplace[] = [
+  {
+    icon: <DiamondIcon />,
+    title: 'Start lending in JewelSwap',
+    description: 'Now you can to lend directly to JewelSwap. Withdraw and Compound are only available after epoch finish, make sure to check that at jewelswap.io.',
+    claimableAmount: 0,
+    imgSrc: xSpotlightLogo,
+    imgComponent: <img height="25" src={xSpotlightLogo} alt="xSpotlight" />,
+  },
+];
```

Comment:
- we should rename the variable "marketplaces" with something custom to JewelSwap and not use a map if you do not plan adding more cards of the same functionality, but just render the card itself
- TypeScript interfaces could also be custom to your needs and not use the NFTMarketplace ones as it might be confusing